controller/array_action/array_mediator_interface.py

```diff
@@ -245,6 +245,8 @@ def create_snapshot(self, volume_id, snapshot_name, pool=None):
             PermissionDenied
             NotEnoughSpaceInPool
             SnapshotSourcePoolMismatch
+            MaximumSnapshotsReached
+            MaximumVolumesReached
```

Comment:
shouldn't `MaximumVolumesReached` also be raised in create volume?

Reply:
Maybe, but it's not related to RE status code.